frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -176,13 +177,15 @@ export const ProjectFeatureToggles = ({
                           header: 'By',
                           cell: ({ row: { original } }) => {
                               return (
-                                  <UserAvatar
+                                  <UserAvatarWithPopover
                                       user={{
                                           id: original.createdBy.id,
                                           name: original.createdBy.name,
                                           imageUrl: original.createdBy.imageUrl,
                                       }}
-                                      avatarWidth={(theme) => theme.spacing(3)}
+                                      avatarWidth={(theme: Theme) =>
+                                          theme.spacing(3)
+                                      }
```

Comment:
My linter was complaining about the "any" type here (probably because of the extra layer of abstraction for the props), so I added it explicitly.